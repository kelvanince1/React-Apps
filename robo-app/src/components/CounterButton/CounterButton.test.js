import React from 'react';
import { shallow } from 'enzyme';

import CounterButton from './index';

it('Expect to render Counter Button component', () => {
    expect(shallow(<CounterButton color="red" />)).toMatchSnapshot();
});

it('Correctly increments the counter', () => {
    const wrapper = shallow(<CounterButton color="red" />);

    wrapper.find('[id="counter"]').simulate('click');
    wrapper.find('[id="counter"]').simulate('click');
    expect(wrapper.state()).toEqual({ count: 2 });
});
