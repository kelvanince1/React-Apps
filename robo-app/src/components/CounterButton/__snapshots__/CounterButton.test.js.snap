// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Card component 1`] = `ShallowWrapper {}`;

exports[`Expect to render Counter Button component 1`] = `ShallowWrapper {}`;
